packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -195,12 +195,12 @@ export class AtomicSearchInterface {
   }
 
   @Listen('atomic/initializeComponent')
-  public handleInitialization(event: InitializeEvent) {
+  public async handleInitialization(event: InitializeEvent) {
```

Comment:
A change from sync to async on a public component method would in theory be a breaking change. The method signature has changed, and the function does not return the same type as before.


I don't think this change in particular will necessarily cause problems for actual implementation, but just a heads up that we have to be generally careful with those.